# HG changeset patch
# User Miguel Bazdresch <lmb@2pif.info>
# Date 1330613939 0
#      Thu Mar 01 14:58:59 2012 +0000
# Branch stable
# Node ID 245963d3d6289934c4d2e7689722aa43fc582359
# Parent  d27e638b1c3aa089a49aebe113cfce34cd2412f0
pkg: bug fix - accessing non-existent variable for error message

diff --git a/scripts/pkg/pkg.m b/scripts/pkg/pkg.m
--- a/scripts/pkg/pkg.m
+++ b/scripts/pkg/pkg.m
@@ -1746,17 +1746,18 @@ function desc = get_description (filenam
       if (length (colon) == 0)
         disp ("skipping line");
       else
         colon = colon(1);
         keyword = tolower (strip (line(1:colon-1)));
         value = strip (line (colon+1:end));
         if (length (value) == 0)
             fclose (fid);
-            error ("the keyword %s has an empty value", desc.keywords{end});
+            error ("The keyword `%s' of the package `%s' has an empty value",
+                    keyword, desc.name);
         endif
         desc.(keyword) = value;
       endif
     endif
     line = fgetl (fid);
   endwhile
   fclose (fid);
 
